grr/server/grr_response_server/gui/ui/components/timestamp/relative_timestamp_pipe.ts

```diff
@@ -0,0 +1,54 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {DateTime} from 'luxon';
+
+@Pipe({
+  name: 'relativeTimestamp'
+})
+export class RelativeTimestampPipe implements PipeTransform {
+  readonly oneDay = 86400000; // One day in millis
+  readonly oneHour = 3600000; // One hour in millis
+  readonly oneMin = 60000; // One minute in millis
+  readonly oneSecond = 1000; // One second in millis
+
+  transform(jsDate: Date, absoluteOnly: boolean): any {
```

Comment:
Is it possible to change the return type from any to string?

Reply:
Sure!